### DISCUSSION TITLE
Port Open Indicator

### DISCUSSION THREAD

**Discussion by ajhorvath20** [jesec/flood#322] [Q&A]
Hi, is it possible to know if the port is open or not. In `qbittorrent `and `rtorrentvpn `there us an icon at the bottom to show port is open/active if its green but I cannot find it anywhere in flood. Maybe I am missing :) 

Thanks

**Comment by jesec** [comment]
A local service has no way to ascertain if it is reachable externally, without the assistance of an external service. 

It is not preferable for Flood to contact an external service unsolicitedly and unnecessarily. 

As a result, unfortunately, I am not going to implement such feature in Flood. Nonetheless, if the API of torrent client exposes such status, I think it is OK for Flood to display it.

**Reply by ajhorvath20** [reply]
Currently I am only `curling` inside the container to get the `ip`. Something like that `docker exec -it rflood curl icanhazip.com`
